### ISSUE TITLE
Support text suggestions for fcitx5-unikey and fcitx5-bamboo

### ISSUE THREAD

**Issue by zenfas** [fcitx/fcitx5-unikey#29]
Currently, fcitx5-unikey and fcitx5-bamboo don't support for Vietnamese text suggestion/prediction.
Can you make it base on database from Dictionary of Ho Ngoc Duc (famous public Vietnamese dictionary on internet) with Viet22K.txt or Viet11K.txt (bigger include some word not use frequently at the moment and size bigger)?

http://www.informatik.uni-leipzig.de/~duc/software/misc/wordlist.html
https://github.com/duyet/vietnamese-wordlist

**Comment by trmdi** [comment]
Unikey on Windows doesn't have this feature too.

**Comment by zenfas** [comment]
Yes, I know. I request new feature like Gboard or Laban Key now do on Android

**Comment by trmdi** [comment]
Can you give an example?

**Comment by zenfas** [comment]
![image001(2)](https://user-images.githubusercontent.com/66258788/200736237-5ab10da6-d201-41d7-a620-804dfd2addd4.png)
I think it good for phrase, example when you're typing "ánh" fcitx will display “ánh/ánh đèn/ánh nắng/ánh sáng” and user can choose

**Comment by trmdi** [comment]
That's good on mobile but I don't like this idea on PC.

**Comment by zenfas** [comment]
If assign good shortcut key to choose candidate, it's a fast and good option for choise phrase. I see on Chinese, they're usually usage. Currently, fcitx5-android support unikey on Android, if have, will be very good.

**Comment by zenfas** [comment]
And we can make it an option, if people like you don't like, you can disable :)
